File: src/Microsoft.AspNetCore.SignalR.Client/HubException.cs
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

using System;

namespace Microsoft.AspNetCore.SignalR.Client
{
    [Serializable]
    public class HubException : Exception
    {
        public HubException()
        {
        }

        public HubException(string message) : base(message)
        {
        }

        public HubException(string message, Exception innerException) : base(message, innerException)
        {
        }
    }
}

File: clients/java/signalr/src/main/java/com/microsoft/signalr/HubException.java
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

package com.microsoft.signalr;

public class HubException extends RuntimeException {
    private static final long serialVersionUID = -572019264269821519L;

    public HubException() {
    }

    public HubException(String errorMessage) {
        super(errorMessage);
    }

    public HubException(String errorMessage, Exception innerException) {
        super(errorMessage, innerException);
    }
}
